### PR TITLE
Update replicate-mongodb.sh to use govuk-env-sync backups

### PR DESCRIPTION
**Pull request by deborahchua**
This script was using backups that are generated specifically for local Docker development under /mongodb/daily/ in the S3 bucket, whereas all other backups use the govuk-env-sync process.

The following have already been migrated:
- maslow_production (DocumentDB)
- short_url_manager_production (DocumentDB)
- govuk_assets_production (DocumentDB)
- travel_advice_publisher_production (DocumentDB)
- govuk_content_production (DocumentDB)
- imminence_production (PostgreSQL)

These apps are the last ones remaining on MongoDB and will be migrated
to DocumentDB or PostgreSQL. authenticating-proxy has not been included
as it is in the process of being migrated - see https://trello.com/c/lSpntlfk/81-mongo-26-has-reached-end-of-life
for more information.

Trello card: https://trello.com/c/mOYRWDOc/3084-modify-govuk-docker-so-that-it-uses-the-govuk-env-sync-backups-3

### CODE REVIEW

**Review by catalinailie — COMMENTED** [review]
Is the `.DS_Store` file added by mistake?

Other than that looks good to me, but I haven't interacted with this so far, so I might not be the best person to review the changes.

File: bin/replicate-mongodb.sh [hunk 1]
```diff
@@ -16,22 +16,22 @@ bucket="govuk-integration-database-backups"
...
-  "manuals-publisher"|"publisher"|"specialist-publisher")
-    hostname=mongo
-    database=govuk_content_production
+  "licensify")
+    hostname=mongo-licensing
+    database=licensify
+    ;;
+  "content-store"|"draft-content-store")
+    hostname=mongo-api
+    database=content_store_production
```
Comment (ChrisBAshton) [inline bin/replicate-mongodb.sh:34]: This should dynamically choose the draft or live backup (like we do for Router above):

```suggestion
    database="${app//-/_}_production"
```

(Also matches line 38)

**Comment by deborahchua** [comment]
> Is the `.DS_Store` file added by mistake?
> 
> Other than that looks good to me, but I haven't interacted with this so far, so I might not be the best person to review the changes.

Ah thanks @catalinailie that file shouldn't be included!

**Review by ChrisBAshton — COMMENTED** [review]
Great digging! +1 to the .DS_Store (I did briefly consider us adding it to [.gitignore](https://github.com/alphagov/govuk-docker/blob/main/.gitignore), but there are [reasons not to do that](https://github.com/alphagov/content-publisher/pull/1744))

**Comment by ChrisBAshton** [comment]
PS: please summarise in this PR which [apps you've ignored](https://trello.com/c/mOYRWDOc/3084-modify-govuk-docker-so-that-it-uses-the-govuk-env-sync-backups-3#comment-63e1375a4e990ca580f36bb1), and why. (I.e. if someone tries replicating authenticating-proxy data before it's finished migrating to PostgreSQL, we should be able to point them to something that explains the situation).

**Comment by deborahchua** [comment]
> PS: please summarise in this PR which [apps you've ignored](https://trello.com/c/mOYRWDOc/3084-modify-govuk-docker-so-that-it-uses-the-govuk-env-sync-backups-3#comment-63e1375a4e990ca580f36bb1), and why. (I.e. if someone tries replicating authenticating-proxy data before it's finished migrating to PostgreSQL, we should be able to point them to something that explains the situation).

Thanks @ChrisBAshton ! I've updated the PR and commit description with more details.

**Review by ChrisBAshton — APPROVED** [review]
Great work ⭐

**Pull request merged**